src/main/resources/anniv5Resources/localization/eng/eurogamepack/Powerstrings.json

```
@@ -0,0 +1,33 @@
+{
+  "${ModID}:QuickGamePower": {
+    "NAME": "Quick Game",
+    "DESCRIPTIONS": [
+      "Entering #yVictory requires half as much #yVP (#b", ")."
```

```suggestion
      "Entering #yVictory requires half as much #yVP ( #b", " )."
```
Otherwise the highlighting would be off